contributor/es/06-conclusion-script-es.asciidoc

```diff
@@ -0,0 +1,35 @@
+# Transcripción: El contribuidor en InnerSource – Introducción 
+
+=== Duración: 1-3 Minutos
+
+=== Actores: Isabel y Johannes
+
+=== Resumen del “PlayBook”
+
+Este segmento recapitula que hemos aprendido acerca de ser un contribuidor InnerSource. Compartiremos como puedes continuar tu aprendizaje en InnerSource, ambos con otros videos en línea y también el involucrarse con la comunidad en línea de InnerSource.
+
+=== Transcripción
+
+J: Gracias por ver el segmento de contribuidor de la ruta de aprendizaje de InnerSource Commons. Con esta sección aprendiste sobre el rol de contribuidor – La sangre vital de los proyectos de InnerSource:
```

Comment:
IIRC, we use to capitalize Contribuidor, as it refers to a specific InnerSource role, beyond the regular meaning of the word.